Hymns/Browse/TagListView.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+import Resolver
+
+struct TagListView: View {
+
+    @ObservedObject private var viewModel: TagListViewModel
+
+    init(viewModel: TagListViewModel = Resolver.resolve()) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        VStack {
+            List(viewModel.tags, id: \.self) { tag in
+                NavigationLink(destination: BrowseResultsListView(viewModel: BrowseResultsListViewModel(tag: tag))) {
+                    Text(tag)
+                }
```

Comment:
I forgot to add a TODO here for the empty state, for when viewModel.tags is empty